### ISSUE TITLE
bigwig.q0.xxx  errors out run ends

### ISSUE THREAD

**Issue by tollecr** [KCCG/ClinSV#66]
When you run the docker version, v1.0:

docker pull mrbradley2/clinsv:v1.0

sampleInfo.txt:
NG130ECKCK	rg	lb	/app/input/NG130ECKCK.bam

I also had to make links to the bam files within the alignments\NG130ECKCK folder  so that bigwig.createWigs.NG130ECKCK.sh could mostly run.

It errors out and if you run it again it always ends at the same place:

docker run --platform linux/amd64 -v $refdata_path:/app/ref-data \
-v $project_folder:/app/project_folder  \
-v $input_path:/app/input  \
--entrypoint "perl" mrbradley2/clinsv:v1.0 /app/clinsv/bin/clinsv \
-r all \
-p /app/project_folder \
-i "/app/input/*.bam" \
-ref /app/ref-data/refdata-b38 \

End of the output obtained:
 
# OK # job bigwig, createWigs, NG130ECKCK ran successfully (exit=0) -> skip : /app/project_folder/alignments/NG130ECKCK/bw/sh/bigwig.createWigs.NG130ECKCK.sh

 ### executing: sh /app/project_folder/alignments/NG130ECKCK/bw/sh/bigwig.q0.NG130ECKCK.sh &> /app/project_folder/alignments/NG130ECKCK/bw/sh/bigwig.q0.NG130ECKCK.e  ...  

 ### finished after (hh:mm:ss): 00:01:56
 ### exist status: 35072

See the file
/app/project_folder/alignments/NG130ECKCK/bw/sh/bigwig.q0.NG130ECKCK.e
below:  

 ***** error exist status != 0 (35072), please check /app/project_folder/alignments/NG130ECKCK/bw/sh/bigwig.q0.NG130ECKCK.e for more information

**/app/project_folder/alignments/NG130ECKCK/bw/sh/bigwig.q0.NG130ECKCK.e :**
+ export PATH=/app/clinsv/bin:/app/clinsv/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+ PATH=/app/clinsv/bin:/app/clinsv/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+ cd /app/project_folder/alignments/NG130ECKCK/bw/
+ wigToBigWig /app/project_folder/alignments/NG130ECKCK/bw/tmp/NG130ECKCK.q0.wig /app/ref-data/refdata-b38/genome/Homo_sapiens_assembly38.fasta.chrom.sizes /app/project_folder/alignments/NG130ECKCK/bw/NG130ECKCK.q0.bw
/app/project_folder/alignments/NG130ECKCK/bw/sh/bigwig.q0.NG130ECKCK.sh: line 7:    36 Killed                  wigToBigWig /app/project_folder/alignments/NG130ECKCK/bw/tmp/NG130ECKCK.q0.wig /app/ref-data/refdata-b38/genome/Homo_sapiens_assembly38.fasta.chrom.sizes /app/project_folder/alignments/NG130ECKCK/bw/NG130ECKCK.q0.bw

issues listed as #58 and #53 don't help solve the issue.  As #58 states I believe there are path issues here.  Not sure how to solve them.  You do need to correct the README, v1.0.0 doesn't exist:

docker pull mrbradley2/clinsv:v1.0

This could be very useful to many if you could get it working for the community/beginners.

**Comment by avp65** [comment]
it seems we have now the same error. I saw on the issue #34  that it may be caused by lack or RAM to run the bigwig step.
For me it makes sense as I am training myself on my personnal computer that have only 16 go of RAM. 
Could you solve your problem ? It was problem with RAM or something else ?

**Comment by J-Bradlee** [comment]
Yes this does look similar to issue #34 . A way to verify this is to look at your OS monitor (task manager or tops etc..) and see if clinsv eats up all the usable ram and then gets shut down

**Comment by tollecr** [comment]
I finally got Clinsv to run properly using docker, once I used hard links to the bam files inside my testing folder, instead of softlinks in macOS.  I thought this might be useful to others.
